I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 473, in script
    $ StartGame()
  File "game/script.rpy", line 473, in <module>
    $ StartGame()
  File "game/game.rpy", line 47, in StartGame
  File "game/game.rpy", line 75, in execute
  File "game/game.rpy", line 75, in execute
  File "game/game.rpy", line 80, in execute
  File "game/game.rpy", line 81, in execute
  File "game/game.rpy", line 82, in execute
Exception: Not a displayable: (700, 350)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 473, in script
    $ StartGame()
  File "D:\renpy-8.1.3-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\renpy-8.1.3-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 473, in <module>
    $ StartGame()
  File "game/game.rpy", line 47, in StartGame
  File "D:\renpy-8.1.3-sdk\renpy\exports.py", line 3347, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "D:\renpy-8.1.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\renpy-8.1.3-sdk\renpy\display\core.py", line 3582, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "D:\renpy-8.1.3-sdk\renpy\display\core.py", line 4055, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "D:\renpy-8.1.3-sdk\renpy\display\core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "D:\renpy-8.1.3-sdk\renpy\display\core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "D:\renpy-8.1.3-sdk\renpy\display\core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "D:\renpy-8.1.3-sdk\renpy\display\screen.py", line 476, in visit_all
    callback(self)
  File "D:\renpy-8.1.3-sdk\renpy\display\core.py", line 4055, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "D:\renpy-8.1.3-sdk\renpy\display\screen.py", line 487, in per_interact
    self.update()
  File "D:\renpy-8.1.3-sdk\renpy\display\screen.py", line 680, in update
    self.screen.function(**self.scope)
  File "game/game.rpy", line 75, in execute
  File "game/game.rpy", line 75, in execute
  File "game/game.rpy", line 80, in execute
  File "game/game.rpy", line 81, in execute
  File "game/game.rpy", line 82, in execute
  File "D:\renpy-8.1.3-sdk\renpy\ui.py", line 955, in _imagebutton
    return renpy.display.behavior.ImageButton(
  File "D:\renpy-8.1.3-sdk\renpy\display\behavior.py", line 1209, in __init__
    idle_=renpy.easy.displayable(idle_image),
  File "D:\renpy-8.1.3-sdk\renpy\easy.py", line 145, in displayable
    raise Exception("Not a displayable: %r" % (d,))
Exception: Not a displayable: (700, 350)

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
gorelov_home 1.0
Sun Nov 12 18:31:08 2023
